pkg/resiliency/wrap.go

```diff
@@ -0,0 +1,68 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Authors of Cilium
+
+package resiliency
+
+import (
+	"errors"
+)
+
+// RetryableErr represents an error that can possibly be retried.
+type RetryableErr struct {
+	error
+
+	Kind Category
+}
+
+// Retry checks whether error could be retried.
+// TODO will need m'o tuning..
```

Comment:
What is the intention regarding this TODO?

Reply:
this has been removed